src/node/miner.h

```diff
@@ -160,6 +160,17 @@ class BlockAssembler
         // Whether to call TestBlockValidity() at the end of CreateNewBlock().
         bool test_block_validity{true};
         bool print_modified_fee{DEFAULT_PRINT_MODIFIED_FEE};
+        /**
+         * The maximum additional weight which the pool will add to the coinbase
+         * scriptSig, witness and outputs. This must include any additional
+         * weight needed for larger CompactSize encoded lengths.
+         */
+        size_t coinbase_max_additional_weight{4000};
+        /**
+         * The maximum additional sigops which the pool will add in coinbase
+         * transaction outputs.
+         */
+        size_t coinbase_output_max_additional_sigops{400};
```

Comment:
I think it should not be necessary to reference pool in here

```suggestion
        /**
         * The maximum additional weight that can be added  to the coinbase
         * scriptSig, witness and outputs. This must include any additional
         * weight needed for larger CompactSize encoded lengths.
         */
        size_t coinbase_max_additional_weight{4000};
        /**
         * The maximum additional sigops that can be added in coinbase
         * transaction outputs.
         */
        size_t coinbase_output_max_additional_sigops{400};
```

Reply:
See above https://github.com/bitcoin/bitcoin/pull/30356#discussion_r1660679371, just omitting the word "pool" because in theory there could be something _other_ than a pool that wants to add data to our coinbase, just seems confusing to me.